src/prognostic_equations/implicit/implicit_solver.jl

```diff
@@ -932,10 +954,11 @@ function update_implicit_equation_jacobian!(A, Y, p, dtγ, t)
             @. ∂ᶜmseʲ_err_∂ᶜq_totʲ =
                 dtγ * (
                     -DiagonalMatrixRow(
-                        adjoint(ᶜinterp(ᶠu³ʲs.:(1))) *
-                        ᶜgradᵥ_ᶠΦ *
-                        Y.c.ρ *
-                        ᶜkappa_mʲ / ((ᶜkappa_mʲ + 1) * ᶜp) * ∂e_int_∂q_tot,
+                        adjoint(ᶜinterp(ᶠu³ʲs.:(1))) * ᶜgradᵥ_ᶠΦ * Y.c.ρ / ᶜp *
+                        (
+                            (ᶜkappa_mʲ / (ᶜkappa_mʲ + 1) * ∂e_int_∂q_tot) +
```

Comment:
```suggestion
                            (ᶜkappa_mʲ / (ᶜkappa_mʲ + FT(1)) * ∂e_int_∂q_tot) +
```

Reply:
@dennisYatunin I don't think I need FT here?

Reply:
No need for FT here, since `+(::AbstractFloat, ::Int)` will always be promoted to an `AbstractFloat`